website/versioned_docs/version-0.4.x/set-up-your-environment.mdx

```diff
@@ -484,14 +501,142 @@ IMPORTANT: Ceramic API running on 0.0.0.0:7007
 
 By this point you should have your development environment and all configurations in place to get started working on your application.
 
+## Frequently Asked Questions
+
+---
+
+Some questions and issues come up more often than others. We've compiled a list of the most common ones here.
+
+<details>
+  <summary>Which setup method is better: wheel or JavaScript package managers?</summary>
+  <div>
+    <div>
+      Wheel is the recommended and the easiest way to configure your working environment and install
+      all the necessary dependencies. We highly recommended going with wheel if you are just
+      starting out with Ceramic. Everything will be taken care of for you.
+    </div>
+  </div>
+</details>
+<details>
+  <summary>Which operating systems are supported?</summary>
+  <div>
+    <div>
+      It's best to run Ceramic and ComposeDB on Linux or a Mac. You can also run it on Windows but
+      you'll have to use WSL2 (Windows Subsystem for Linux). See the supported operating systems
+      section at the top.
+    </div>
+  </div>
+</details>
+
+<details>
+  <summary>Which node version is preferred?</summary>
+  <div>
+    <div>
+      We have seen the best results using Node.js v16. Earlier or later versions can cause issues
+      for some users. While we're working on eliminating the issues, it's best to use Node v16 for
+      now.
+    </div>
+  </div>
+</details>
+<details>
+  <summary>How long does it take to install the packages?</summary>
+  <div>
+    <div>
+      Installing everything (either with wheel or JavaScript packages) takes usually between 2 and 5
+      minutes. Throughout the guide above you can find what kind of output you should be looking for
+      to know that everything was installed correctly.
+    </div>
+  </div>
+</details>
+<details>
+  <summary>Where in the system do I run all of the commands?</summary>
+  <div>
+    <div>
+      Sometimes, especially when using JavaScript package managers to install Ceramic and ComposeDB,
+      it's easy to forget that you need to run all of the commands in the app's directory. With
```

Comment:
Maybe let's clarify that this directory can be manually created by the developer working on the app? Wheel technically creates a project directory as part of the setup process

Reply:
Added.